src/components/HelpDrawer/HelpContent/CatalogHistogramPlotHelpContent.tsx

```diff
@@ -1,21 +1,23 @@
 export const CATALOG_HISTOGRAM_PLOT_HELP_CONTENT = (
     <div>
         <p>
-            The catalog histogram plot widget shows a histogram of a selected numeric column from a catalog file. The available numeric columns for plotting are determined by the <code>Display</code> column of the upper table in the Catalog
-            widget.
+            The Catalog Histogram Plot Widget offers a dynamic visualization of a histogram derived from a chosen numeric column within a catalog file. The available numeric columns for plotting are determined by the "Display" column of the
+            upper table within the Catalog Widget.
         </p>
         <p>
-            The data used for plotting the histogram is determined by the lower table in the Catalog widget. The table may not show all the entries because of the progressive loading feature. When you scroll down the catalog table, new
-            catalog entries will be streamed for visualization. Thus, the histogram plot may not include all entries (even after filtering) when the widget is initialized. The <code>Plot</code> button allows you to request a full download
-            of all the entries and the histogram plot will then include all data (after filtering). The number of bins and the scale of the y-axis (linear or log) can be customized.
+            The data used for plotting a histogram is determined by the lower table within the Catalog Widget. Due to the progressive loading mechanism, not all entries may be immediately visible within the table. As you navigate through
+            the catalog table, new entries are progressively loaded to facilitate table visualization. Consequently, it's important to note that when initializing the widget, the histogram plot might not encompass all entries (even
+            post-filtration). The <b>Plot</b> button enables you to trigger a full download of all entries. This ensures that the histogram plot incorporates the complete dataset post-filtration. Flexibility is granted in terms of selecting
```

Comment:
Initially, the frontend will only receive the first 50 entries of the catalog. If now we plot histogram or 2D scatter, only these 50 entries of data will be used to generate the plot. By clicking the "Plot" button, all the other data will then be sent to the frontend. See the following screenshot as the demo. See the text info at the bottom of the lower table for the numbers.

However, think this behavior again, we probably should plot all (or all after filtering) data at the time when the histogram/scatter plot widget is initialized so the behavior is consistent with the image overlay rendering. (This can be a post-v4 issue)

<img width="715" alt="Screenshot 2023-08-28 at 08 42 39" src="https://github.com/CARTAvis/carta-frontend/assets/20819712/9901f78b-2226-437b-ae49-22a2cd740f55">
<img width="718" alt="Screenshot 2023-08-28 at 08 42 48" src="https://github.com/CARTAvis/carta-frontend/assets/20819712/91caa9ff-5e23-4b61-893d-ba1dcd528f63">

Reply:
Oh, I see there are two "Plot" buttons, one in the catalog widget and another one in the histogram/scatter plot widget. Because we talk about tables before the description, and then upper part of the histogram/scatter plot widget after the description, I thought it's the "Plot" button in the catalog widget. Perhaps change the content to "The <b>Plot</b> button in the Histogram/Scatter Plot Widget."

Reply:
good point. fixed.